src/components/LoginHero/LoginHero.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useRef } from 'react';
+import './LoginHero.scss';
+
+const d = 300
+
+function randomDist(dist) {
+    return (Math.random() * 2 - 1) * dist
+}
+
+export default function LoginHero() {
+    let messages = ["Hello Nimbus", "Welcome to CSAI", "Check us out on Github"];
+    let index = Math.floor(Math.random() * messages.length)
+    let messageRef = useRef(null)
+    let currentMessage = messages[index].split('').map((char, i) => <span className='letter' style={{
+        display: `${char === ' ' ? 'initial' : ''}`
+    }} key={i} > {char}</ span >)
+
+    let animateLetters = () => {
+        let letters = [...messageRef.current.children]
+        letters.forEach(letter => {
+            // animate each individual span element (character)
+            let keyframes = {
+                transform: [`translate(${1 * randomDist(d)}px, ${1 * randomDist(d)}px)`, 
+                `translate(${0.8911 * randomDist(d)}px, ${0.8911 * randomDist(d)}px)`,
+                `translate(${0.5644 * randomDist(d)}px, ${0.5644 * randomDist(d)}px)`,
+                `translate(${0.0299 * randomDist(d)}px, ${0.0199 * randomDist(d)}px)`,
+                `translate(${0.0066 * randomDist(d)}px, ${0.0066 * randomDist(d)}px)`,
+                `translate(${0.0163 * randomDist(d)}px, ${0.0163 * randomDist(d)}px)`,
+                `translate(${0.0154 * randomDist(d)}px, ${0.0154 * randomDist(d)}px)`,
+                `translate(0, 0)`],
+                opacity: ['0.2', '1']
+            }
+            letter.animate(keyframes, { duration: 4000, easing: 'cubic-bezier(0.37, 0, 0.63, 1)' })
```

Comment:
More of a note of something to try. Add a `delay` property to the AnimationOptions based on the character index in the forEach loop. Something like:
```
{delay: i * 100}
```
Sequencing the letters in this way might look kinda cool!

Reply:
I had this idea too! I didn't know about the delay property, thanks for this tip. I implemented it with [1cf0af9](https://github.com/calpoly-csai/nimbus-validator-app/pull/27/commits/1cf0af95b6b4439bcb50c25b243f085866248abd), let me know if you guys like it.